platform/example.anchor-config.yaml

```diff
@@ -70,23 +70,24 @@ app-config:
     #     STRICT_SEND: amount_in = amount
     #     STRICT_RECEIVE: amount_in = amount + fee
     paymentType: STRICT_SEND
+    depositInfoGeneratorType: self # self or circle
```

Comment:
Can we have some comments for `self`?

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/234/commits/178cf4281c87d08ea762b15a71c0215aa6d29d81.